app/controllers/ems_common.rb

```diff
@@ -1005,6 +1006,7 @@ def process_emss(emss, task)
           :target_class => model.to_s)
     elsif task == "destroy"
       model.where(:id => emss).order("lower(name)").each do |ems|
+        ems.disable_ems
```

Comment:
Sounds like it's still a race whether workers exit before destroy gets executed.
We also wanted to delay the actual destruction until all workers actually die, no?  (probably by polling whether workers exist & re-queuing if not?)